app/build.gradle

```diff
@@ -128,7 +128,7 @@ android {
         targetSdkVersion 28
         multiDexEnabled true
         versionCode 1500
-        version "2.7.0-dev"
```

Comment:
Usually merging recent state of target banch (dev) before PR and resolving conflicts solves issue. Maybe it is artiffact of switching repos. But since current dev has `-dev` prefix it seems like here as an some strange merge error.